censys/asm/events.py

```diff
@@ -0,0 +1,98 @@
+"""
+Class for interfacing with the Censys Logbook API.
+"""
+from typing import Optional, Generator, List
+from datetime import datetime
+
+
+class Events:
```

Comment:
This should be a child of `CensysAsmAPI` for it to access its own `self._get()` and other `protected members`.